system_tests/features/get_info_endpoints.feature

```diff
@@ -1,16 +1,25 @@
 Feature: Get info endpoints
 
   Scenario Outline: System hits service get info endpoint
-    Given the system requests <service> endpoint info
-    When  the <service> endpoint info is returned
+    Given services are running
+    When  the system requests <service> endpoint info
     Then   a success status code (200) is returned
```

Comment:
Looks like the then step is missing in info_endpoint.py

Reply:
The then step is in the common.py